pkg/controller/topo/entity/reconciler.go

```diff
@@ -90,6 +90,15 @@ func (r *Reconciler) Reconcile(request reconcile.Request) (reconcile.Result, err
 }
 
 func (r *Reconciler) reconcileCreate(entity *v1beta1.Entity) (reconcile.Result, error) {
+
+	// Get topo service name
+	var topoServiceName string
```

Comment:
@kuujo I guess we can set the default serviceName to topoService in crd schema, and remove this part as well.

Reply:
Did this work? So none of the defaults work?

Reply:
@kuujo I hadn't tested this until now, I thought it wouldn't work like the status, but it works. So I updated the controller to get the service name from entity.